apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -28,73 +28,81 @@ var templates = map[string]string{
 	"lowercase": "{{ .name | lower }}",
 }
 
-// ServiceBindingSpec defines the desired state of ServiceBinding
+// ServiceBindingSpec defines the desired state of ServiceBinding.
 type ServiceBindingSpec struct {
-	// MountPath is the path inside app container where bindings will be mounted
-	// If `SERVICE_BINDING_ROOT` env var is present, mountPath is ignored.
-	// If `SERVICE_BINDING_ROOT` is absent and mountPath is present, set `SERVICE_BINDING_ROOT` as mountPath value
-	// If `SERVICE_BINDING_ROOT` is absent but mounthPath is absent, set   SERVICE_BINDING_ROOT as `/bindings`
-	// When mountPath is used, the file will be mounted directly under that directory
-	// Otherwise it will be under `SERVICE_BINDING_ROOT`/<SERVICE-BINDING-NAME>
+	// MountPath specifies the path inside the app container where bindings
+	// will be mounted.  The environment variable `SERVICE_BINDING_ROOT`
+	// has higher precedence than this option, and setting it within an
+	// application to be bound will cause MountPath to be ignored and will
+	// always mount resources as files.  If neither MountPath nor
+	// `SERVICE_BINDING_ROOT` are set, this field defaults to `/bindings`.
+	// This results in the file being mounted under the directory
+	// specified.
 	// +optional
 	MountPath string `json:"mountPath,omitempty"`
 
-	// NamingStrategy defines custom string template for preparing binding names.
-	// It can be pre-defined strategies(i.e none,uppercase), in case strategy provided in this field isn't defined
-	// we are going to treat the value as a custom template and prepare binding names accordingly.
+	// NamingStrategy defines custom string template for preparing binding
+	// names.  It can be set to pre-defined strategies: `none`,
+	// `lowercase`, or `uppercase`.  Otherwise, it is treated as a custom
+	// go template, and it is handled accordingly.
 	// +optional
 	NamingStrategy string `json:"namingStrategy,omitempty"`
 
-	// Custom mappings
+	// Mappings specifies custom mappings.
 	// +optional
 	Mappings []Mapping `json:"mappings,omitempty"`
 
-	// Services is used to identify multiple backing services.
+	// Services indicates the backing services to be connected to by an
+	// application.  At least one service must be specified.
 	// +kubebuilder:validation:MinItems:=1
 	Services []Service `json:"services"`
 
-	// Application is used to identify the application connecting to the
-	// backing service operator.
+	// Application identifies the application connecting to the backing
+	// service.
 	Application Application `json:"application"`
 
-	// DetectBindingResources is flag used to bind all non-bindable variables from
-	// different subresources owned by backing operator CR.
+	// DetectBindingResources is a flag that determines if the subresources
+	// of a backing operator custom resource will be bound or not.
```

Comment:
subresources will not be bound - if true, SBO will try to find binding data in resources owned by the specified service resource.